applications/MetisApplication/custom_processes/metis_divide_heterogeneous_input_in_memory_process.h

```diff
@@ -131,11 +130,10 @@ class MetisDivideHeterogeneousInputInMemoryProcess : public MetisDivideHeterogen
      */
     void Execute() override
     {
-        int mpi_rank;
-        int mpi_size;
+        MPI_Comm this_comm = MPIDataCommunicator::GetMPICommunicator(mrDataComm);
```

Comment:
also the use of MPIDataComm is currently enforces, this is the only way to get the raw MPI_Comm

but then again, I will refactor/remove this asap